include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -79,7 +79,15 @@ class UTVariadicRTLOp<string mnemonic, list<OpTrait> traits = []> :
                !listconcat(traits,
                            [SameTypeOperands, SameOperandsAndResultType])> {
 
+  let hasCanonicalizer = 1;
   let hasFolder = 1;
+  let verifier = [{
+    auto size = inputs().size();
+    if (size < 1)
+      return emitOpError("requres 1 or more args");
```

Comment:
Or better yet, this should be a generic constraint.  Can you propose something in mlir/include/mlir/IR/OpBase.td?

Reply:
Variadics can have 1 argument, but this case is eliminated by folder